signinapp/events.py

```diff
@@ -51,6 +52,12 @@ class EventForm(FlaskForm):
     end = DateTimeLocalField(format=DATE_FORMATS)
     type_id = SelectField(label="Type", choices=lambda: get_form_ids(EventType))
     enabled = BooleanField(default=True)
+    funds = DecimalField(label="Funds Received", filters=[lambda x: int(x * 100)])
```

Comment:
These filters seem to cause the funds to grow every time you edit the page (they also currently break creating an event because it tries to apply the filter to a `None` value.